base/arraymath.jl

```diff
@@ -204,7 +204,7 @@ end
 """
     rotl90(A, k)
 
-Rotate matrix `A` left 90 degrees an integer `k` number of times.
+Left rotate matrix `A` 90 degrees counter-clockwise an integer `k` number of times.
```

Comment:
```suggestion
Rotate matrix `A` 90 degrees counterclockwise an integer `k` number of times.
```
Counterclockwise is one word.